helpers/AppNavigation.js

```diff
@@ -2,6 +2,8 @@ import { createStackNavigator, createAppContainer } from 'react-navigation';
 import HomeScreen from '../screens/HomeScreen';
 import LoginScreen from '../screens/LoginScreen';
 import SplashScreen from '../screens/SplashScreen';
+import ContactsScreenStyles from '../styling/ContactsScreen.styles';
```

Comment:
I would move the styles into a file next to the component like so: 
![Screenshot 2019-07-18 at 09 31 33](https://user-images.githubusercontent.com/29771398/61438044-db128000-a93e-11e9-8f1c-99d444d91363.png)

That way you don't have to navigate to the other side of the project just to find the component's styles. Also, keep in mind that this falls under the umbrella of writing your components in a modular way.

Reply:
Will do! Thank you